src/link_manager/LinkManagerStateMachine.cpp

```diff
@@ -879,15 +881,15 @@ void LinkManagerStateMachine::handleSwitchActiveRequestEvent()
 }
 
 // 
-// ---> handleDefaultRouteStateNotification(const std::string &routeState);
+// ---> handleDefaultRouteStateNotification(const DefaultRoute routeState);
 // 
 // handle default route state notification from routeorch
 //
-void LinkManagerStateMachine::handleDefaultRouteStateNotification(const std::string &routeState)
+void LinkManagerStateMachine::handleDefaultRouteStateNotification(const DefaultRoute routeState)
 {
-    MUXLOGWARNING(boost::format("%s: state db default route state: %s") % mMuxPortConfig.getPortName() % routeState);
+    MUXLOGDEBUG(mMuxPortConfig.getPortName());
```

Comment:
The new log message is almost useless?

Reply:
> The new log message is almost useless?

I used this kind of log to confirm notification was passed from `DbInterface` all the way to `LinkManagerStateMachine`, and handlers were executed.